webapp/components/new-test-results-history-grid.js

```diff
@@ -231,17 +234,22 @@ class TestResultsGrid extends PolymerElement {
   }
 
   // get test history and aligned run data
-  async getTestHistory() {
-    if (!this.path) {
+  async getTestHistory(path) {
+    if (!path) {
       throw new Error('Test path is undefined');
     }
```

Comment:
This check is redundant now that the value is checked for in `displayCharts()`, so this block can probably just be removed

Reply:
done 👍